vnext/Microsoft.ReactNative/XamlHelper.idl

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.ReactNative {
   runtimeclass XamlHelper {
```

Comment:
>XamlHelper [](start = 15, length = 10)

Ideally we must not have such class and instead implement ReadValue and WriteValue for the Windows.UI.Color in the shared Cxx code.

Reply:
Added ReadValue for Color in JSValueXaml.h like Brush does, so we can just do `To<Color>`, but I guess we still need keep the helper class public for Cxx project to use, otherwise we'd have to move all the underlying util functions/classes over.

Reply:
Could you use the To<Color> in the custom view for now and I will revisit the XamlHelper next week to see how we can make it better. One of my main concerns is that the Color and Brush must be serialized from IJSValueReader and not from IJSValueWriter. I guess we will need to change the underlying implementation to use IJSValueReader instead of folly::dynamic.

Reply:
Sounds good, thanks!